Header header
geometry_msgs/Point maxPoint
geometry_msgs/Point minPoint
geometry_msgs/Point32[] worldPos
std_msgs/Float32MultiArray occupancyGrid
